org.contourterminal.Contour.yml

```diff
@@ -0,0 +1,91 @@
+app-id: org.contourterminal.Contour
```

Comment:
Just clarify, contourterminal.org is an actual domain you own?

Reply:
I bought it this weekend exclusively for the App-Id, yes.